app/views/hiring_staff/vacancies/job_summary/show.html.haml

```diff
@@ -8,7 +8,7 @@
 .govuk-grid-row
   .govuk-grid-column-two-thirds
     = form_for @job_summary_form, url: school_job_job_summary_path(@vacancy.id), html: { method: "patch" }, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f|
-      = render 'hiring_staff/vacancies/error_messages', errors: @job_summary_form.errors
+      = f.govuk_error_summary t('jobs.errors_present')
```

Comment:
👍 nice!

Reply:
I briefly played around with setting this message as default in the config, but had no success